#BIOL366 
## What Do We Expect?
- **Sequencing methods**
	- **Sanger**
	- Pyro
	- Illumina 
- **Chemical Synthesis of DNA**
	- Steps
- extraction of dna
- **Microarrays**
	- Steps
- **Hybridization**
	- **Melting point calculation**
- **Photolithography**
- Blotting Methods
	- Southern -> DNA
	- Northern -> RNA

## Sample Questions (Set 1)

### 1 C
DNA$\rightarrow$RNA$\rightarrow$Protein

### 2 A
The template DNA strand is read in the 3’ to 5’ direction, the mRNA is created in the 5’ to 3’ direction

### 3 A
**A-U** H-bonds

No *hydrogen bonds* between A-A and G-U, however these interactions are present.

### 4 D
All of the above.

### 5 B
I. The empty tRNA exits the ribosome as the ribosome moves along the mRNA.  
II. A tRNA charged with an amino acid binds to the complementary codon on the mRNA.  
III. The ribosome catalyzes the transfer of the growing polypeptide from the tRNA bound  
in the P site to the amino acid on the tRNA in the A-site.  
IV. A ribosome binds to an mRNA transcript.  
V. The ribosome moves along the mRNA and the complete polypeptide is released.

**Order:** IV, II, III, I, V

### 6 C

### 7 C
DNA methylation *is* present in plants.

### 8 D
**Order**: IV, III, I, II  

### 9 B
B-DNA

**A-DNA** is *very tightly coiled.* RNA is A. *C3 endo.*
**B-DNA** is *normal*. *C2-endo.*
**Z-DNA** is distorted and weird. Impure with a lot of ions and -OH groups in weird positions.
### 10 Guess D -> its A
Double check. Process of elimination.

### 11 B


### 12 B
*They are secondary structures.*

### 13 C


### 14 A


### 15 B


### 16 C


### 17 D


### 18 B


### 19 D


### 20 C


## Sample Questions (Set 2)

### 1 
It means that the phosphate on the 5' end of the next nucleotide can only be added to the free 3'-OH group of the last nucleotide after the removal of a pyrophosphate.

### 2 
Methylation is common at the C5 position of cytosine, to produce 5-methylcytosine at CpG, CpHpG, and CpHpH sites. This is important for regulation of chromosome stability, and a potential marker of disease states.

mRNA can also be methylated at U or A, which is important to regulate stability and translation. Also a marker of disease states.

### 3 
Higher GC = higher melting point. There are more H bonds.

### 4 
**||** $\rightarrow$ **|**| + |**|**
Half of every DNA molecule is comprised of a completely newly synthesized strand of DNA and a previously synthesized strand of DNA.

### 5 
Chemical synthesis of DNA. *ITS BACKWARDS*
1. Bind first base to silica support.
2. Deprotect DMT
3. Next nucleotide is added via activated and protected phosphate group at the 3' position.
4. Oxidize Phosphate
5. Repeat 1-3 for however many *nt*s you need.
6. Deprotect DMT
7. Deprotect cyanoethyl
8. Remove from silica support

### 6 
Much like proteins, the folding of RNA occurs spontaneously because it is more thermally stable. Therefore, 3d structures such as advanced hairpins form much more readily in RNA than in DNA. The 2' OH group is quite reactive, the missing methyl group on uracil (compared to cytosine) and has a more compact A-DNA structure.

### 7 
Whatever

### 8 
^^^

### 9 
Primary - AA order (sequence).
Secondary - Interactions between the backbone ($\alpha$-helix and $\beta$-sheets).
Tertiary - Interactions between side-chains (disulfide bridges, ion-ion interactions, hydrophobic interactions).
Quaternary - same as tertiary but between polypeptide chains.

**Replacement of Proline**: 
![[Drawing 2023-09-24 19.37.36.excalidraw]]

Proline is usually present at the end of an $\alpha$-helix. Therefore, this will definitely cause a disruption in the secondary structure of the protein.

Cysteine is responsible for the formation of disulphide bridges, so these interactions would be missing, inevitably changing the tertiary or quaternary structure of the protein.

### 10 
Whatever

### 11 
Whatever

### 12 

### 13 

### 14 

### 15 

### 16 

### 17 
GC content,
Length of the sequence,
Salt concentration,
DNA concentration.

### 18 


### 19 
Denaturation **95**$\degree$
Annealing (Primers) **58**-**72**$\degree$
Elongation **72**$\degree$

### 20 

## Sanger Sequencing Problems

### 1. 
**Sequence dsDNA**
```
5’-ATCGGGCTAGCTGACTTTACC (N460) ATCGGGCTAGCTGACTTTACC-3’
3'-CGACCCGATCGACTGAAATGG (N460) CGACCCGATCGACTGAAATGG-5'
```

**Forward Primer**
`5'-ATCGGGCTAGCTGACTTTACC-3'`

**Reverse Primer**
`5'-GGTAAAGTCAGCTAGCCCAGC-3'`

### 2.
**Microarray Experiment**
Procure a microarray that contains randomised but known primers (oligos) in each cell.
Using chemistry, fluorescently tag the extracted cDNA (produced from RNA).
Run the cDNA on the microarray, and put it together like a puzzle :) Fun with friends :)

### 3.


## Chapter 6 Problems
**#2, 3, 4, 5, 9, 10, 12, 13, 14**
## 2
N3 and N11 are acceptor